src/main/java/com/minecolonies/coremod/entity/ai/util/Structure.java

```diff
@@ -152,14 +152,15 @@ else if (structureBlock instanceof BlockStairs && structureBlockState.equals(wor
             {
                 return true;
             }
-            else if((structureBlock == Blocks.DIRT || structureBlock == Blocks.GRASS) && (worldBlock == Blocks.DIRT || worldBlock == Blocks.GRASS))
+            else if ((structureBlock == Blocks.DIRT || structureBlock == Blocks.GRASS) && (worldBlock == Blocks.DIRT || worldBlock == Blocks.GRASS))
             {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') This branch's code block is the same as the block for the branch on line 152. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1871)